tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: prepare-repo
+      image: quay.io/konflux-ci/release-service-utils:4576e1e2a30439129cb69c8a4f39f7ced2d44376
+      script: |
+        set -eux
+
+        MRRC_FILE="$(workspaces.data.path)/$(params.mrrcParamFilePath)"
+        . "$MRRC_FILE"
+        echo "Downloading the maven repo zip $MRRC_ZIP_REGISTRY"
+        mrrc_dir="$(dirname $MRRC_FILE)/mrrc"
+        mkdir -p $mrrc_dir
+        cd $mrrc_dir
+        oras pull $MRRC_ZIP_REGISTRY
```

Comment:
So this is all happening in the shared workspace, right? I think that might be problematic, because the PVC is only something like 1gb. Normally I would recommend using a temp dir (preferably via `mktemp -d`). So either you can use that and have just one step. Or else you can use an emptyDir workspace in the Pod and then you can access it in both steps. See for example here: https://github.com/konflux-ci/release-service-catalog/blob/7f1b6ba9ee108d1cd9973b1d6ab9eb182fa3aff3/tasks/push-rpm-manifests-to-pyxis/push-rpm-manifests-to-pyxis.yaml#L35

Reply:
Ok. Will look at it.

Reply:
Changed accordingly in new commit.